design/prototypes/chrome-extension/src/styles/global.css

```
@@ -0,0 +1,672 @@
+  100% {
+    opacity: 0;
+    transform: translate(-50%, -40px) scale(1.08);
+  }
+}
+
+.sr-status {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)